talon/quotations.py

```diff
@@ -204,6 +204,8 @@ def mark_message_lines(lines):
     markers = ['e' for _ in lines]
     i = 0
     while i < len(lines):
+        if ignore_initial_spaces:
+            lines[i] = lines[i].lstrip(' ')
```

Comment:
I'd rather use some local variable for the line and not change the list of lines since lists are mutable. In general I'd say if it's not required for a parameter to be changed inside of a function it's better not to change it. Sometimes it's clear from a function definition / signature that a certain parameter is gonna change inside. Otherwise you have this ambiguity and some might assume that the parameter is gonna remain unchanged.